DOCUMENTATION.md

```diff
@@ -0,0 +1,271 @@
+
+- Git operations module consists of three basic git operations( Stage all changes, commit changes, push to remote ). Below this file status is display and the files can be staged with add action or click on "stage all changes" to stage all the untracked files from the chosen repo.
+
+  
+
+![Picture7](https://user-images.githubusercontent.com/65342122/87232645-30f8f700-c3de-11ea-8ddb-52f4d5ec7140.png)
+
+- After add operation all staged files can be removed immediately if you wish to, else ignore it. Reload the page to see the difference in "File View".
+
+![Picture8](https://user-images.githubusercontent.com/65342122/87232658-51c14c80-c3de-11ea-95e1-b9bbeeac82bb.png)
+
+- All staged files can be committed using "Commit changes" module. After commit changes reload the page to see active branch name, number of tracked files and number of commits.
+
+  
+
+![Picture9](https://user-images.githubusercontent.com/65342122/87232659-56860080-c3de-11ea-9bc4-a19ad727b101.png)
+
+- After **commit changes** operation,
+
+  
+
+![Picture10](https://user-images.githubusercontent.com/65342122/87232661-5c7be180-c3de-11ea-82b7-104792c2e3ec.png)
+
+  
+
+- "Push to remote" module pushes all commits to the remote host. If there is no remote in the chosen repo then ignore this module. After the push to remote is completed, check the remote host to see the pushed folder/file.
+
+with Remote(Git Hub)
+
+![Picture11](https://user-images.githubusercontent.com/65342122/87232662-61409580-c3de-11ea-8ad7-61c3871f0a4d.png)
+
+If you try to "push to remote" without having a remote host, "push failed" error message will be displayed.
+
+  
+
+![Picture12](https://user-images.githubusercontent.com/65342122/87232666-6867a380-c3de-11ea-9903-5ea12200e994.png)
+ 
+
+In "Git Difference" click on the modified file(s) to see the difference.
+
+![Picture13](https://user-images.githubusercontent.com/65342122/87232669-6d2c5780-c3de-11ea-9739-f8181e4d0901.png)
+
+- "Stage all changes" and "commit changes" for applying a change.
+
+  
+
+- Settings in the left pane has three sections( Server data file, saved repos, Active GitConvex port number ).
+
+  
+
+![Picture15](https://user-images.githubusercontent.com/65342122/87232673-75849280-c3de-11ea-9d01-fe3479282561.png)
+
+  
+
+- Server data file stores repo details. The data file can be updated. The data file must be an accessible JSON file with read / write permissions set to it. Also make sure you enter the full path for the file. E.g: /opt/my_data/data-file.json
+
+- In the saved repos section, added repo(s) can be deleted permanently from GitConvex.
+
+- The port number can be updated but make sure to restart the app and to change the port in the URL after updating it.
+
+- Visit help section if you're facing an issue or need any help. If you have any queries discuss it in "Discord" or report an issue in GitHub.
+
+![Picture16](https://user-images.githubusercontent.com/65342122/87232674-79b0b000-c3de-11ea-904a-c6a02c4d15f5.png)
```

Comment:
The image is too congested. Leave adequate margin spacing while capturing the snapshot